site2/docs/schema-understand.md

```diff
@@ -539,8 +539,18 @@ This diagram illustrates how does schema work on the Producer side.
 3. The broker looks up the schema in the schema storage to check if it is already a registered schema. 
    
 4. If yes, the broker skips the schema validation since it is a known schema, and returns the schema version to the producer.
+
+5. If no, the broker verifies whether a schema can be automatically created in this namespace:
+
+  * If `isAllowAutoUpdateSchema` sets to **true**, then a schema can be created, and the broker validates the schema based on the schema compatibility check strategy defined for the topic.
+  
+  * If `isAllowAutoUpdateSchema` sets to **false**, then a schema can not be created, and the consumer is rejected to connect to the broker.
   
-5. If no, the broker validates the schema based on the schema compatibility check strategy defined for the topic. 
+**Tip**:
+
+`isAllowAutoUpdateSchema` can be set via **Pulsar admin API** or **REST API.** 
+
+For how to set `isAllowAutoUpdateSchema` via Pulsar admin API, see [Manage AutoUpdate Strategy](schema-manage.md/#manage-autoupdate-strategy). 
```

Comment:
Seems a little weird between #5 and #6? Do we need to add some contents like "If a schema is updated automatically:” after #5?